.github/workflows/beam_PreCommit_Website.yml

```diff
@@ -0,0 +1,49 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: beam_PreCommit_Website
+
+on:
+  push:
+    tags: ['v*']
+    branches: ['master', 'release-*']
+    paths: ['website/**']
+  pull_request:
+    branches: ['master', 'release-*']
+    paths: ['website/**']
+  issue_comment:
+    types: [created]
+  schedule:
+    - cron: '* */6 * * *'
+
+jobs:
+    beam_PreCommit_Website:
+        if: ${{github.event.issue.pull_request}} || ${{github.event.comment.body == 'Run Website PreCommit'}} || ${{github.event.schedule}}
+        runs-on: [self-hosted, ubuntu-20.04]
+        name: beam_PreCommit_Website
+        steps:       
+          - name: Git checkout
+            uses: actions/checkout@v3
+          - name: Install Java
```

Comment:
Same as https://github.com/apache/beam/pull/27099#discussion_r1231104791, we might need define a single "install java" (or self hosted image has pre-installed java) in the future